internal/manifests/manifests.go

```diff
@@ -42,8 +44,14 @@ func BuildAll(params Params) ([]client.Object, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	querierObjects := querier.BuildQuerier(params.Tempo)
+	queryFrontendObjects := queryfrontend.BuildQueryFrontend(params.Tempo)
```

Comment:
if these two do not return error then the call can be inlined into append like for the gossip objects.